docs/getting-started.md

```diff
@@ -550,10 +558,22 @@ json.dump(forecast, sys.stdout)
 
 To write the data loader in R, name it <code>forecast.json.R</code>. Or as shell script, <code>forecast.json.sh</code>. You get the idea. See [Data loaders: Routing](./loaders#routing) for more. The beauty of this approach is that you can leverage the strengths (and libraries) of multiple languages, and still get instant updates in the browser as you develop.
 
-### Deploying automatically
+### Deploy automatically
 
 You can schedule builds and deploy your project automatically on commit, or on a schedule. See [deploying](./deploying) for more details.
 
+### Update Framework
+
+To update Framework to the latest [release](https://github.com/observablehq/framework/releases), cd into the project directory and run:
+
+<pre data-copy>npm update <span class="win">"</span>@observablehq/framework<span class="win">"</span></pre>
+
+or
+
+<pre data-copy>yarn upgrade <span class="win">"</span>@observablehq/framework<span class="win">"</span></pre>
```

Comment:
Since Framework has a fair number of dependencies, would it be better to have people run a bare `npm update` instead of just updating the `@observablehq/framework` package? I also find that if you just upgrade packages individually, you often get multiple versions of packages, and hence it’s generally better to upgrade everything together.